components/general/SendMsg.tsx

```diff
@@ -0,0 +1,123 @@
+import styles from "@/styles/ContactPage.module.css";
+import { fontSyne, fontRobotoMono } from "@/pages/_app";
+import SocialMedia from "./SocialMedia/SocialMedia";
+import Button from "../base/Button";
+import { useEffect, useState } from "react";
+
+const SendMsg = () => {
+  const [isNameFieldEmpty, setIsNameFieldEmpty] = useState(true);
+  const [isEmailFieldEmpty, setIsEmailFieldEmpty] = useState(true);
+  const [isMessageFieldEmpty, setIsMessageFieldEmpty] = useState(true);
+  const handleNameFieldChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setIsNameFieldEmpty(event.target.value.length == 0 ? true : false);
+  };
+  useEffect(() => {
+    const label = document.getElementById("username_label");
```

Comment:
Do you think you can use useRef instead of getElementById?

Reply:
OK, I will try that